inst/odin/lancelot.R

```diff
@@ -20,16 +20,27 @@ update(time) <- (step + 1) * dt
 ## output number of individuals vaccinated by age and vaccine stage
 ## For example, for E, we sum over n_E_next_vacc_class (those moving vaccine
 ## stage without progressing disease stages)
-## vaccinated S
-n_S_vaccinated[, ] <- n_S_next_vacc_class[i, j]
+## Note that individuals making vaccine skip moves are counted in all classes
```

Comment:
not sure what this comment means @edknock ?

Reply:
ah now I remember this discussion - worth expanding a bit I think to explain this is only for the output it doesn't affect the model dynamics

Reply:
I've expanded this